static/app/components/events/interfaces/spans/spanTree.tsx

```diff
@@ -881,9 +881,13 @@ function SpanRow(props: SpanRowProps) {
 }
 
 const TraceViewContainer = styled('div')`
-  overflow-x: hidden;
+  overflow-x: visible;
   border-bottom-left-radius: 3px;
   border-bottom-right-radius: 3px;
+  > div,
+  > div > div {
+    overflow: visible !important;
+  }
```

Comment:
```suggestion
  overflow-x: hidden;
  border-bottom-left-radius: 3px;
  border-bottom-right-radius: 3px;
```
I don't think we can actually commit these changes as is right now, it actually impacts the page height for normal span tree pages. This was just a hack to get the span frequency percentages showing next to each span row. We can probably revert this part for now until we find a better way to do the span frequency css without affecting page height.